wp/lib/bap_wp/src/precondition.mli

```diff
@@ -60,7 +65,8 @@ val get_vars : Bap.Std.Sub.t -> Bap.Std.Var.Set.t
     given the name of each variable. *)
 val get_output_vars : Bap.Std.Sub.t -> string list -> Bap.Std.Var.Set.t
 
-(** Create a Z3 expression that denotes a load in memory [mem] at address [addr]. *)
+(** Create a Z3 expression that denotes a load in memory [mem] at address [addr]
```

Comment:
Can we describe each input in the comments, even if the are somewhat self evident?